pkgs/tools/video/yamdi/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  fetchurl,
+  ...
```

Comment:
You can omit the dots here. These are used to match all remaining attributes in an attribute set. This is needed if your package is called with a fixed set of arguments or when you don't know all inputs/only want to pick needed ones. `callPackage` however will only pass needed packages. You can further put this in one line.